tests/testCourseElementExtension.js

```diff
@@ -0,0 +1,123 @@
+const assert = require('chai').assert;
+const { step } = require('mocha-steps');
+const fs = require('fs-extra');
+const config = require('../lib/config');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const sqlLoader = require('@prairielearn/prairielib/sql-loader');
+const sql = sqlLoader.loadSqlEquiv(__filename);
+const _ = require('lodash');
+const path = require('path');
+const freeform = require('../question-servers/freeform.js');
+const { promisify } = require('util');
+
+const helperServer = require('./helperServer');
+const helperClient = require('./helperClient');
+
+describe('Course element extensions', function() {
+    this.timeout(60000);
+    const exampleCourseDir = path.join(__dirname, '..', 'exampleCourse');   
```

Comment:
I'm worried that we're re-growing the "Example Course" to a "Test Course" status. Maybe guidelines should be drawn up on this? (Not a show stopper here... More thinking out loud.)